templates/reqpage.go

```diff
@@ -76,6 +76,7 @@ const RequestPage = `<!DOCTYPE html>
 								<li>Did you upload all data?</li>
 								<li>Does your repository contain a comprehensive description of the data (preferably in the README.md file)?</li>
 								<li>Does your repository contain a LICENSE file with a license matching the one indicated in datacite.yml?</li>
+								<li>Does your repository contain code or content licensed under different terms? You can include a separate LICENSE file for parts of the repository and describe its application in the README.</li>
```

Comment:
```suggestion
								<li>Does your repository contain code or content licensed under different terms? Please include a separate LICENSE file for parts of the repository and describe its application in the README.</li>
```

Reply:
I would not option them to include an additional license if it is required but request it instead.

Reply:
Great point.